DIMS-Core.Database/DIMS-Core.Database/Views/vUserTask.sql

```diff
@@ -0,0 +1,7 @@
+create or alter view vUserTask
+as
+select UserProfiles.UserId, Task.TaskId, Task.Name as TaskName, Task.Description, Task.StartDate, Task.DeadlineDate, TaskState.StateName
```

Comment:
You can user UserId column from UserTask table so you don't have to join UserProfiles table